docs/how-to/use-layers.md

```diff
@@ -0,0 +1,147 @@
+# How to use layers
+
+Managing multiple services across different environments becomes complex as systems scale. Pebble simplifies this with layered configurations, improving readability and maintainability.
+
+A base layer defines common settings (such as logging), while additional layers handle specific services or environment-specific overrides. This declarative approach, along with delegated layer management, allows for better cross-team collaboration and provides a clear view of each environment's configuration. For example, an operations team could manage base layers for logging, and service teams could manage layers for their services.
+
+## Pebble layers
+
+A layer is a configuration file that defines the desired state of services running on a system. 
+
+Layers are organized within a `layers/` subdirectory in the `$PEBBLE` directory. Their filenames are similar to `001-base-layer.yaml`, where the numerically prefixed filenames ensure a specific order of the layers, and the labels after the prefix uniquely identifies the layer. For example, `001-base-layer.yaml`, `002-override-layer.yaml`.
+
+A layer specifies service properties including the command to execute, startup behavior, dependencies on other services, and environment variables. For example:
```

Comment:
This is still too service-specific. A layer can describe service properties as well as health checks and log targets.